### PR TITLE
cleaned settings menu. edited bubby credit

### PR DESCRIPTION
**Pull request by SecondaryProfile**
Wrapped elements in Sized box. Potentially can abstract this in the future...

### CODE REVIEW

File: creaphur/lib/screens/dashboard/settings.dart [hunk 1]
```diff
@@ -56,28 +56,41 @@ class SettingsScreen extends StatelessWidget {
...
               onChanged: (String? val) => selectProfile(val!)),
           const SizedBox(
-            height: 12,
+            height: 22,
           ),
-          FilledActionButton(
-            buttonText: 'Add Profile',
-            onPressed: goToProfile,
+          SizedBox(
+            width: 800,
```
Comment (jcoolu) [inline creaphur/lib/screens/dashboard/settings.dart:62]: Can we change the width here to `width: double.infinity` instead?

File: creaphur/lib/screens/dashboard/settings.dart [hunk 2]
```diff
@@ -56,28 +56,41 @@ class SettingsScreen extends StatelessWidget {
...
+              type: FileType.custom,
+              childWidget: const Text('Import Save Data'),
+            ),
           ),
-          FilledActionButton(
-            onPressed: () async => Utils.exportSaveData(context),
-            buttonText: 'Export Save Data',
+          SizedBox(height: 20),
+          SizedBox(
+            width: 800.0,
```
Comment (jcoolu) [inline creaphur/lib/screens/dashboard/settings.dart:88]: Can we change the width here to `width: double.infinity` instead?

File: creaphur/lib/screens/dashboard/settings.dart [hunk 3]
```diff
@@ -56,28 +56,41 @@ class SettingsScreen extends StatelessWidget {
...
-            height: 15,
+            height: 39,
           ),
-          OutlinedFilePicker(
-            onChange: (contents) async =>
-                Utils.importSaveData(contents, context),
-            type: FileType.custom,
-            childWidget: const Text('Import Save Data'),
+          SizedBox(
+            width: 800,
```
Comment (jcoolu) [inline creaphur/lib/screens/dashboard/settings.dart:77]: Can we change the width here to `width: double.infinity` instead?

File: creaphur/lib/screens/dashboard/settings.dart (hunk 2 above)
Comment (SecondaryProfile) [reply creaphur/lib/screens/dashboard/settings.dart:88]: if you want the buttons to fit the screen, I thought the width was a little below that threshold but I can test to see if that is comparable

File: creaphur/lib/screens/dashboard/settings.dart (hunk 3 above)
Comment (SecondaryProfile) [reply creaphur/lib/screens/dashboard/settings.dart:77]: Reference above comment

File: creaphur/lib/screens/dashboard/settings.dart (hunk 1 above)
Comment (SecondaryProfile) [reply creaphur/lib/screens/dashboard/settings.dart:62]: Reference below comment

File: creaphur/lib/screens/dashboard/settings.dart (hunk 2 above)
Comment (jcoolu) [reply creaphur/lib/screens/dashboard/settings.dart:88]: Dew it

**Pull request merged**